notebooks/fashionmnist_autoencoder.livemd

```diff
@@ -0,0 +1,145 @@
+# Fashion mnist autoencoder
+
+## Introduction
+
+Autoencoder is a deep learning model which consists of two parts: encoder and decoder. The role of the encoder is to compress high dimensional data into low dimensional representation. Then decoder tries to recreate the original data from the encoded one.
+Autoencoders architecture is widely used in the following problems:
+
+* Dimensionality reduction
+* Noise reduction
+* Generative models
+* Data augmentation
+
+To learn more about autoencoders visit [an article about autoencoders](https://towardsdatascience.com/applied-deep-learning-part-3-autoencoders-1c083af4d798 "an article about autoencoders") and [Wikipedia](https://en.wikipedia.org/wiki/Autoencoder "Wikipedia") .
```

Comment:
Should I replace these links with other resources or completely drop this part? Maybe this would be better https://web.stanford.edu/class/psych209a/ReadingsByDate/02_06/PDPVolIChapter8.pdf (very first article about autoencoders).